README

```diff
@@ -12,10 +12,10 @@ applications communicate to form a single distributed application.
 
```

Comment:
The changes in README look good.  Can we push this in a separate bug fix/commit?